packer/build_image.sh

```diff
@@ -257,11 +257,11 @@ if [ "$TARGET" = "aws" ]; then
     arch="$ARCH"
     case "$arch" in
       "x86_64")
-        SOURCE_AMI_FILTER="ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-amd64*"
+        SOURCE_AMI_FILTER="ubuntu-minimal/images/hvm-ssd/ubuntu-jammy-22.04-amd64*"
```

Comment:
Q: Is that a stable image? I wasn't sure if the minimal image is stable, or changes regularly. I want to ensure we have a reproducible from known good source build!

Reply:
@fruch Those kernel issues were mainly around Centos7 which used an old one. I can't remember we had such an issue with Ubuntu. also we build new images on a daily basis, our kernel version can be different between each build , so nothing changed

Reply:
> @fruch Those kernel issues were mainly around Centos7 which used an old one. I can't remember we had such an issue with Ubuntu. also we build new images on a daily basis, our kernel version can be different between each build , so nothing changed 

If it did happen, doesn't mean it couldn't happen. 

The issue with CentOS7 wasn't that it was old, it was giving us new kernels that wasn't tested by AWS. And new canonical are working closly with AWS to test it, still doesn't mean it could break for some of our use cases.

Having reproducible build has it's own virtue, and I still think the release branches should be a moving target 🎯